pkgs/applications/science/math/mathematica/l10ns.nix

```diff
@@ -8,31 +8,34 @@ let allVersions = with lib; flip map
   # N.B. Versions in this list should be ordered from newest to oldest.
   [
     {
-      version = "12.3.1";
+      version = "13.0.0";
```

Comment:
I think the point of having multiple options is to accommodate people owning licenses for different versions.

Reply:
The last several version updates in this file followed the same process, and it appears that `default.nix` doesn't pass in the version so it'll always will be the latest for a given language.

I'll go through the Git history and add back the last several previous versions that were removed.

Reply:
I've added a commit that adds them back.